### PR TITLE
net-proxy/sqtop: initial version.

### PR DESCRIPTION
**Pull request by jkroonza**
sqtop is like top, but for squid.

It connects to squid's manager object to retrieve information about
what's going on, and then visually displays the information.

Closes: https://bugs.gentoo.org/414577
Package-Manager: Portage-2.3.89, Repoman-2.3.20
Signed-off-by: Jaco Kroon <jaco@uls.co.za>

### CODE REVIEW

File: net-proxy/sqtop/Manifest [hunk 1]
```diff
@@ -0,0 +1 @@
+DIST v2015-02-08.tar.gz 154535 BLAKE2B 4825a254670e5336acbddc8ffb50c7a423614c06fcf9b69b354e3ccf88496bde17ba586bf45cf062a4da40e2332b84a611b50582e425278a259cc27a23f53f7f SHA512 35911b2d4aaa6364ec52bb55264372378b4a458268cfc2e03884660576014739a55e255eb751a9be54b70d4504f16d095b6cf9db08ff5889d52fd2a46ab2033a
```
Comment (juippis) [inline net-proxy/sqtop/Manifest]: So this is going to divide people and I'm on the side who don't wish "new" packages to be last-released in 2015. Maybe GURU is a better place for this for now?

File: net-proxy/sqtop/sqtop-20150208.ebuild [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PV="${PV:0:4}-${PV:4:2}-${PV:6:2}"
```
Comment (juippis) [inline net-proxy/sqtop/sqtop-20150208.ebuild]: I'd rename this 2015.02.08 and you can use a simple `ver_rs` snippet here.

File: net-proxy/sqtop/sqtop-20150208.ebuild [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+DESCRIPTION="Squid Top - top for Squid"
+HOMEPAGE="https://github.com/paleg/sqtop"
+SRC_URI="https://github.com/paleg/sqtop/archive/v${MY_PV}.tar.gz"
+
+S="${WORKDIR}/${PN}-${MY_PV}"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
```
Comment (juippis) [inline net-proxy/sqtop/sqtop-20150208.ebuild]: Empty variable can be removed.

File: net-proxy/sqtop/Manifest (hunk 1 above)
Comment (jkroonza) [reply net-proxy/sqtop/Manifest]: Package is stable, fit for purpose.  Does what it does well and unless something changes squid side no changes should be required.  Only breakage in the history of this use was a minor tweak required from squid3 to squid4 - else you'd probably be looking at a 2007 package.

We find this extremely useful.  Others may or may not.

File: net-proxy/sqtop/sqtop-2015.02.08.ebuild [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PV="$(ver_rs 1-2 -)"
+
+DESCRIPTION="Squid Top - top for Squid"
+HOMEPAGE="https://github.com/paleg/sqtop"
+SRC_URI="https://github.com/paleg/sqtop/archive/v${MY_PV}.tar.gz"
```
Comment (juippis) [inline net-proxy/sqtop/sqtop-2015.02.08.ebuild:10]: `-> ${P}.tar.gz` for an identifiable distfile.

Comment (jkroonza) [reply net-proxy/sqtop/sqtop-2015.02.08.ebuild:10]: added locally.  i'm able to reproduce ncurses but haven't dug into it yet.

**Pull request closed without merging**